iquip/monitor.py

```diff
@@ -0,0 +1,112 @@
+"""Module for monitoring devices."""
+
+from typing import Any, TypeVar, Generic, Optional, Callable
+
+from PyQt5.QtWidgets import QWidget, QLabel, QHBoxLayout
+from PyQt5.QtCore import pyqtSignal
+
+
```

Comment:
How about removing a blank line?
Of course, we put two lines after import in `qiwis.py`, but only one line in most other codes such as examples of `qiwis` and `explorer.py`.